### PR TITLE
[feat-11] linkTester file added concatURL and linkPinger working

### PR DESCRIPTION
**Pull request by mj-e**
please check functions 
cheers boss

### CODE REVIEW

File: initialUrl.json [hunk 1]
```diff
@@ -1 +0,0 @@
-http://www.msn.com
```
Comment (Willpicky16) [inline initialUrl.json:1]: What is this?

File: linkTester.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+var https = require('https');
+
+function concatUrl(arr) {
+    const ncLink = 'https://broken-links-api.herokuapp.com';
+    return arr.map(function (x) {
+        return /https/.test(x) ? x : ncLink + x;
+    });
+}
+
+function linkTester(links) {
```
Comment (Willpicky16) [inline linkTester.js:12]: I like this function 🗡

File: linkTester.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+// to be finished
```
Comment (Willpicky16) [inline linkTester.js:13]: lol

**Review by Willpicky16 — APPROVED** [review]
I like your work Matt - boss is very happy!

File: initialUrl.json (hunk 1 above)
Comment (mj-e) [reply initialUrl.json:1]: unnecessary code

File: linkTester.js (hunk 2 above)
Comment (mj-e) [reply linkTester.js:12]: thanks for the feedback

File: initialUrl.json (hunk 1 above)
Comment (Willpicky16) [reply initialUrl.json:1]: 🤳

**Pull request merged**